resources/stateresourcemanager.go

```diff
@@ -27,24 +27,21 @@ var resourceRegistry = map[string]reflect.Type{
 	AutoSubnetResource: reflect.TypeOf(AutoSubnetCfgResource{}),
 }
 
-// EtcdResourceManager implements the core.ResourceManager interface.
-// It manages the resource in a logically centralized manner using serialized
-// writes to a etcd based datastore.
-type EtcdResourceManager struct {
-	//XXX: should be '*drivers.EtcdStateDriver', but leaving is
-	//core.StateDriver to get tests going and until the netmaster
-	//is changed to pickup the resource-manager from config
-	Etcd core.StateDriver
+// StateResourceManager implements the core.ResourceManager interface.
+// It manages the resources in a logically centralized manner using serialized
+// writes to underlying state store.
+type StateResourceManager struct {
+	StateDriver core.StateDriver
```

Comment:
A style nitpick but perhaps `NewStateResourceManager(stateDriver)` would allow us to set this up as an interface, which should make it easier to extend.